scripts/readme.md

```diff
@@ -1,15 +1,9 @@
-# Local Build and Testing
```

Comment:
This is already in the [CONTRIBUTING.md doc](https://github.com/ionic-team/stencil/blob/master/.github/CONTRIBUTING.md#testing-changes-against-a-project-locally)